core-bundle/src/Controller/InitializeController.php

```diff
@@ -83,7 +83,7 @@ public function __invoke(): InitializeControllerResponse
     {
         trigger_deprecation('contao/core-bundle', '4.0', 'Using custom entry points has been deprecated and will no longer work in Contao 5.0.');
 
-        $masterRequest = $this->requestStack->getMasterRequest();
+        $masterRequest = $this->requestStack->getMainRequest();
```

Comment:
```suggestion
        $mainRequest = $this->requestStack->getMainRequest();
```

Reply:
Fixed in 46c12f9d122c998a7f7e24d7e0e39ad8c0aa693f